ablateLibrary/finiteVolume/processes/twoPhaseEulerAdvection.hpp

```diff
@@ -21,6 +33,8 @@ class TwoPhaseEulerAdvection : public Process {
     };
     Parameters parameters;
 
+    static PetscErrorCode FormFunction(SNES snes, Vec x, Vec F, void* ctx);
```

Comment:
Is this right? It looks like you changed this name.

Reply:
I just moved the function declaration to live within the TwoPhaseEulerAdvection to cleanup the auto docs generated.